tests/testsuite/kubevirtresource.go

```diff
@@ -173,7 +173,7 @@ func UpdateKubeVirtConfigValue(kvConfig v1.KubeVirtConfiguration) *v1.KubeVirt {
 		return kv
 	}
 
-	Expect(CurrentSpecReport().IsSerial).To(BeTrue(), "Tests which alter the global kubevirt configuration must not be executed in parallel, see https://onsi.github.io/ginkgo/#serial-specs")
+	Expect(CurrentSpecReport().IsSerial || CurrentSpecReport().IsInOrderedContainer).To(BeTrue(), "Tests which alter the global kubevirt configuration must not be executed in parallel, see https://onsi.github.io/ginkgo/#serial-specs")
```

Comment:
Can we have similar check in the global `AfterEach`?

Reply:
I do not understand this check/condition.
Ordered tests can run in parallel to other tests and it is fine. They cannot run in parallel to the one in their own group, which is part of ginkgo property, we do not need to interfere and add any asserts for it.

If I missed something, please be patient with me :)

Reply:
> I don't understand that either. I added Ordered because it failed to cleanup the configuration when I tested it with `Ordered`.

You are claiming here that we should not touch the KV config in the ordered containers. That does not make sense. You can change it, as long as you return it for the next test if relevant.
You cannot depend on the "cleanup" to do it for you.

If you have something specific that fails without this, lets investigate and understand it well.